content/pages/homepage/index.json

```diff
@@ -60,9 +60,14 @@
       "challenges/133-time-tables-cardioid-visualization/showcase/contribution8.json",
       "challenges/133-time-tables-cardioid-visualization/showcase/contribution11.json"
     ],
-    "cta": {
+    "featuredCta": {
       "text": "Ready to be inspired?",
       "buttonText": "Enjoy this showcase highlight"
+    },
+    "showcaseCta": {
```

Comment:
@klinegareth I think only one of these CTAs is needed on the homepage at the moment, unless you were working on a revised passenger showcase section.